scripts/publish_python.js

```diff
@@ -69,18 +64,11 @@ const dist_folders = [
 
 // Artifacts inside those folders
 const wheels = [
-    // Mac 10.15
-    // "cp36-cp36m-macosx_10_14_x86_64",
-    "cp37-cp37m-macosx_10_15_x86_64",
-    "cp38-cp38-macosx_10_15_x86_64",
-    "cp39-cp39-macosx_10_15_x86_64",
-
-    // // Mac 11
-    // // NOTE: not yet
-    // // "cp36-cp36m-macosx_11_0_x86_64",
-    // "cp37-cp37m-macosx_11_0_x86_64",
-    // "cp38-cp38-macosx_11_0_x86_64",
-    // "cp39-cp39-macosx_11_0_x86_64",
+    // Mac 11
+    // "cp36-cp36m-macosx_11_0_x86_64",
+    "cp37-cp37m-macosx_11_0_x86_64",
```

Comment:
Just note that the Mac OS version and wheel version are not necessarily equal, I believe we'll still be making 10.15 wheels on OS X 11 so maybe just tweak the comments